modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -85,7 +85,14 @@
 		_p(1,'</PropertyGroup>')
 	end
 
-
+--
+-- Write the available configurations to have correct configuration mapping on vs2022 format and later.
+--
+	function dotnetbase.projectConfigurations(prj)
```

Comment:
That function is never called.

Reply:
i know, i don't kown where to put so it gets placed in the primary property group.